### PR TITLE
Update PayFineUI.java

### PR DESCRIPTION
**Pull request by dipanshu42**

### CODE REVIEW

File: library/payfine/PayFineUI.java [hunk 1]
```diff
@@ -1,27 +1,27 @@
-package library.payfine;
+package library.PayFine;                               //payfine
```
Comment (dharam4) [inline library/payfine/PayFineUI.java:1]: The package  name should be as it is.

File: library/payfine/PayFineUI.java [hunk 2]
```diff
@@ -1,27 +1,27 @@
-package library.payfine;
+package library.PayFine;                               //payfine
 import java.util.Scanner;
 
 
 public class PayFineUI {
```
Comment (dharam4) [inline library/payfine/PayFineUI.java:5]: change the  PayFineUI into PayFineUi @dharam4

File: library/payfine/PayFineUI.java [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
 
 	
-	public PayFineUI(pAY_fINE_cONTROL control) {
-		this.CoNtRoL = control;
+	public PayFineUI(PayFineControl control)       //pAY_fINE_cONTROL control{
+		this.control = control;                //CoNtRoL
 		input = new Scanner(System.in);
-		StAtE = uI_sTaTe.INITIALISED;
-		control.SeT_uI(this);
+		StAtE = UiState.INITIALISED;           //uI_sTaTe.INITIALISED
```
Comment (dharam4) [inline library/payfine/PayFineUI.java:18]: Change the StAtE  into  state @dharam4

**Comment by dipanshu42** [comment]
merging with the master branch

**Pull request merged**